### PR TITLE
Add wgCommentStreamsEnableSearch to ManageWikiSettings

### PR DESCRIPTION
**Pull request by AgentIsai**
Requested on Discord

### CODE REVIEW

**Review by Universal-Omega — CHANGES_REQUESTED** [review]
The configuration needs added to LocalSettings.php also.

File: ManageWikiSettings.php [hunk 1]
```diff
@@ -419,6 +419,15 @@
 		'help' => 'Enable in talk namespaces',
 		'requires' => [],
 	],
+	'wgCommentStreamsEnableSearch' => [
+		'name' => 'CommentStreams Enable Search',
+		'from' => 'commentstreams',
+		'type' => 'check',
+		'overridedefault' => true,
+		'section' => 'discussion',
+		'help' => 'Allow comments and their titles to appear in search results and search auto-complete',
```
Comment (Universal-Omega) [inline ManageWikiSettings.php:428]: Minor, but maybe some punctuation here?

**Comment by CrystalClearGal** [comment]
For some reason, the config shows in ManageWiki sometimes (as default enabled), allows me to toggle it off, but on save kicks the error:
The following errors occurred:
No changes were made.

And then the line and toggle for this new config disappear from the form altogether. And other times it will just not appear there at all.

**Comment by Universal-Omega** [comment]
> For some reason, the config shows in ManageWiki sometimes (as default enabled), allows me to toggle it off, but on save kicks the error: The following errors occurred: No changes were made.
> 
> And then the line and toggle for this new config disappear from the form altogether. And other times it will just not appear there at all.

Thank you for mentioning. Investigating.

**Pull request merged**